deploy/helm/kubecf/templates/diego-cell-service.yaml

```diff
@@ -0,0 +1,81 @@
+{{- if and .Values.features.multiple_cluster_mode.multiple_cluster.enabled .Values.features.multiple_cluster_mode.diego_segment.enabled }}
```

Comment:
Would `diego-cell-isolation-segment-services.yaml` be a better name for this?

Reply:
okay, the name will be better